conda_forge_tick/utils.py

```diff
@@ -543,3 +544,27 @@ def as_iterable(iterable_or_scalar):
         return iterable_or_scalar
     else:
         return (iterable_or_scalar,)
+
+
+def mamba_cant_solve(packages, os_arch):
+
+    subprocess_list = ['mamba', 'install'] + packages + ['-c', f'https://conda.anaconda.org/conda-forge/{os_arch}', '--override-channels']
```

Comment:
There is a different way to set arches via env vars, but this is probably easier

Reply:
I was looking for that in the docs, couldn't find it :(